Add Image comparison plugin (#56)
digitalocean/do-markdownit: fixtures/full-input.md

- Pass `light` or `dark` to switch the theme of the embed (e.g. `[twitter https://twitter.com/MattIPv4/status/1576415168426573825 dark]`)
- Pass `left`, `center`, or `right` to align the embed (e.g. `[twitter https://twitter.com/MattIPv4/status/1576415168426573825 left]`)

### Image compare

> Compare two images side by side. (url1, url2, height, width)

Reviewer: ```suggestion
Compare two images side by side (url1, url2, height, width):
```

Author: Updated.

> [image_compare https://rb.gy/jykhuo https://rb.gy/zt5afg]

Reviewer: ```suggestion
[image_compare https://assets.digitalocean.com/banners/python.png https://assets.digitalocean.com/banners/javascript.png]
```

- Add `left`, `center`, or `right` to set the alignment of the embed (default is `left`).
- Pass `caption` to include caption under the post (e.g. `[instagram https://www.instagram.com/p/CkQuv3_LRgS caption]`)

### Image compare

Compare two images side by side (url1, url2, height, width):

[compare https://assets.digitalocean.com/banners/python.png https://assets.digitalocean.com/banners/javascript.png]

> _Both the width and height are optional, with the defaults being 500 for both._

Reviewer: ```suggestion
_Both the width and height are optional, with the defaults being 480 and 270 respectively._
```

Author: I missed them too :face_exhaling: 
Updated now.

> _Both the width and height are optional, with the defaults being 500 for both._\

Reviewer: ```suggestion
_Both the width and height are optional, with the defaults being 500 for both._
```
Hard linebreak here seems wrong?

Author: Removed.
